src/server/fmd/licence/section.test.js

```diff
@@ -14,17 +15,30 @@ describe('LicenceSection', () => {
     expect(LicenceSection.config.summaryLink).toBe(checkAnswersPage.urlPath)
   })
 
-  it('should have the correct first page for milk', () => {
+  it('should have the correct first page for milk being moved by prodcuer', () => {
     expect(
       LicenceSection.firstPageFactory({
         about: {
           ...aboutSectionCompleteOtherMovement,
-          whatIsMoving: 'milk'
+          whatIsMoving: 'milk',
```

Comment:
we have data for this one already in that same test-helpers file: aboutSectionCompleteMilkProducer I think it is called